litecli/packages/completion_engine.py

```diff
@@ -110,7 +110,7 @@ def suggest_special(text):
     if cmd in ["\\f", "\\fs", "\\fd"]:
         return [{"type": "favoritequery"}]
 
-    if cmd in ["\\d", "\\dt", "\\dt+", ".schema"]:
+    if cmd in ["\\d", "\\dt", "\\dt+", ".schema", ".import"]:
```

Comment:
Is this the right place for the `.import` statement? 

I would have thought that it had to be **filename** and table/view/schema.

Reply:
Yes, the usage of `.import` is `.import filename table`. For `filename`, I am not sure if there is a way to provide some suggestions for `filename`. I put it here, just want to provide suggestions for `table` parameter. Do you have any suggestion for this case?

Reply:
Yes. Take a look at the `.open` special command. I believe that provides a filename suggestion.

Reply:
@amjith That looks cool, thanks. I didn't notice this.

Reply:
Will you be updating this PR with the filename suggestion?

Reply:
Yes, I will update it and also add tests for `.import` command. ETA: by end of this week.

Reply:
Updated the filename suggestion.